ansible/roles/csm.sbps.dns_srv_records/README.md

```diff
@@ -0,0 +1,37 @@
+csm.sbps.dns_srv_records
+========================
+
+Configure SBPS DNS A and SRV records
+
+Requirements
+------------
+
+None.
+
+Role Variables
+--------------
+
+Dependencies
+------------
+
+Example Playbook
+----------------
+
+```yaml
+    - hosts: Iscsi_Targets
```

Comment:
Should this be `Iscsi_Targets` or `Management_Worker` ?
If this is supposed to run on all NCN workers and nowhere else, then shouldn't we use the latter?